src/astronomy/moshier/diurnal.js

```diff
@@ -39,24 +39,23 @@ $ns.diurnal.aberration = function (last, ra, dec, result) {
 
 /* Diurnal parallax, AA page D3
  */
-$ns.diurnal.parallax = function (last, ra, dec, dist, result) {
+$ns.diurnal.parallax = function (last, ra, dec, dist) {
```

Comment:
Similar recursion confirmation check needed

Reply:
yes, I search all source files for the string "parallax". Scanning these results can quickly allow the review of any uses of this function.